components/bl-stepper-component/src/subcomponents/stepper-item.js

```diff
@@ -0,0 +1,47 @@
+import { StepTitle } from './step-title';
+import { CustomizedStepTitle } from './customized-step-title';
+import { StepperLine } from './stepper-line';
+
+export const StepperItem = ({
+  customized,
+  completedSteps,
+  currentStep,
+  steps,
+  stepIndex,
+  step,
+  stepperClassName,
+}) => {
+  let stepperLineClass = stepperClassName.line;
```

Comment:
`const classes = useLineClasses(stepperClassNames, step.completed)`

and below the component

```
const useLineClasses = (stepperClasses, completed) => {
  const classes = [stepperClassName.line]

  if (completed) {
    classes.push(stepperClasses.lineActive)
  }

  return classes
}
```

Reply:
same for other places with classes